src/layout/Table/config.ts

```diff
@@ -0,0 +1,62 @@
+import { CG } from 'src/codegen/CG';
+import { CompCategory } from 'src/layout/common';
+
+export const Config = new CG.component({
+  category: CompCategory.Form,
+  capabilities: {
+    renderInTable: false,
+    renderInButtonGroup: false,
+    renderInAccordion: false,
+    renderInAccordionGroup: false,
+    renderInCards: false,
+    renderInCardsMedia: false,
+    renderInTabs: true,
+  },
+  functionality: {
+    customExpressions: false,
+  },
+})
+  .extends(CG.common('LabeledComponentProps'))
+  .extendTextResources(CG.common('TRBLabel'))
+  .addProperty(new CG.prop('title', new CG.str()))
+  .addDataModelBinding(CG.common('IDataModelBindingsSimple'))
```

Comment:
This should absolutely be a `group` ~~or `list`~~ binding, not a `simpleBinding` (the latter is used for strings, numbers, simple values - not groups/arrays).

EDIT: Not `list` either, when I thought more about it. In short, we've historically used:
- `simpleBinding` for simple leaf-values like strings, numbers, dates, booleans. This name is just a historical artifact and should be phased out IMO (we should use keys like `value` or `text` in components like `Input`, not `simpleBinding`). 
- `list` points to a an array containing simple values (i.e. `string[]`, `number[]`, `boolean[]`)
- `group` points to an array of objects (`object[]`)

Reply:
Ill have a look at group binding :) Didnt know simpleBinding was going away, but that makes sense :)

Reply:
It's probably not going away any time soon.. :cry: But I consider it left over from times past, back when it was difficult to introduce new data model binding keys, so it seemed natural to just re-use the same keys for everything. Now it's very easy to introduce new bindings, so there's no need to use `simpleBinding` for everything anymore.

Reply:
Is the "group" name special or could it be anything like `tableData`?

Reply:
Not really special, no - `tableData` could work as well. But I always try to go for deduplicating names (so `simpleBinding` -> `simple`, as all keys in the `dataModelBindings` are bindings), so in this case I would just call it `table` (as all all keys in `dataModelBindings` point to data).